invariant.ts

```diff
@@ -314,22 +341,31 @@ export const checkInvariants = (
               );
             }
           } catch (error: any) {
-            // If the function call fails with a runtime error, log a dimmed
-            // message. Since the public function result is ignored, there's
-            // no need to throw an error.
-            radio.emit(
-              "logMessage",
-              dim(
-                `₿ ${simnet.burnBlockHeight.toString().padStart(8)} ` +
-                  `Ӿ ${simnet.blockHeight.toString().padStart(8)}   ` +
-                  `${sutCallerWallet}        ` +
-                  `${targetContractName} ` +
-                  `${underline(selectedFunction.name)} ` +
-                  printedFunctionArgs
-              )
-            );
+            // If the pre dialer fails, throw an error.
+            if (error.message.startsWith("Error in pre dialer")) {
```

Comment:
This can be fragile if a single character or casing changes in this message. Thoughts?